python_bindings/bindings.cpp

```diff
@@ -0,0 +1,1144 @@
+#include "agent.hpp"
+#include "agent_io.hpp"
+#include "agents/activity_agent.hpp"
+#include "agents/discrete_vector_agent.hpp"
+#include "agents/inertial_agent.hpp"
+#include "agents/simple_agent.hpp"
+#include "config_parser.hpp"
+#include "model.hpp"
+#include "model_factory.hpp"
+#include "models/ActivityDrivenModel.hpp"
+#include "models/DeGroot.hpp"
+#include "models/DeffuantModel.hpp"
+#include "models/InertialModel.hpp"
+#include "network.hpp"
+#include "network_generation.hpp"
+#include "network_io.hpp"
+#include "simulation.hpp"
+#include "util/erfinv.hpp"
+#include "util/math.hpp"
+#include "util/misc.hpp"
+#include "util/tomlplusplus.hpp"
+
```

Comment:
I would prefer having these rewritten to clarify the source. i.e. `seldon/agent.hpp` etc.

Might need a PR to `seldon`.